common/solar_system_initializers/HFE_TSP_initializer_thoth_system.txt

```diff
@@ -0,0 +1,369 @@
+			orbit_distance = 8
+			orbit_angle = { min = 90 max = 270 }
+			size = 8
+		}
+	}
+	change_orbit = 40
+	planet = {
+		name = "Elvis 1"
+		class = "pc_asteroid"
+		orbit_distance = 0
+		orbit_angle = random
+		size = 6
+		has_ring = no
+	}
+	planet = {
+		name = "Elvis 2"
+		class = "pc_asteroid"
+		orbit_distance = 0
+		orbit_angle = random
+		size = 4
+		has_ring = no
+	}
+	planet = {
+		name = "Elvis 3"
+		class = "pc_asteroid"
+		orbit_distance = 0
+		orbit_angle = random
+		size = 5
+		has_ring = no
+		init_effect = {
+			if = {
+				limit = { has_global_flag = HFE_TSP_spawn_test }
+				log = "Thoth inner planets and asteroids spawned"
+			}
+		}
+	}
+	change_orbit = 30
+	planet = {
+		name = "Ankh"
+		class = "pc_shrouded"
+		orbit_distance = 5
+		orbit_angle = { min = 90 max = 270 }
+		size = 36
+		has_ring = yes
+#		change_orbit = 5
+		init_effect = {
+			save_global_event_target_as = hfe_thoth_location
+			set_planet_flag = thoth_was_here
+			set_planet_flag = call_thoth
+			if = {
+				limit = { has_global_flag = HFE_TSP_spawn_test }
+				log = "Thoth hiding spot planet spawned"
+			}
+			prevent_anomaly = yes
+			create_country = {
+				name = "Station (Probably Gigantic) AI"
+				type = hfe_thoth
+				name_list = cybrxkhan_fantasy_egyptian
+				released_from_country = event_target:hfe_egypt_country
+				authority = auth_dictatorial
+				ethos = {
+					ethic = ethic_pacifist
+					ethic = ethic_spiritualist
+					ethic = ethic_xenophile
+				}
+				civics = {
+					civic = civic_free_haven
+					civic = civic_ancient_machine_intelligence
+#					civic = civic_military_order
+				}
+				origin = hfe_civic_egypt_rulers_origin
+				flag = {
+					background = {
+						category = backgrounds
+						file = "new_dawn.dds"
+					}
+					icon = {
+						category = corporate
+						file = "corporate_06.dds"
+					}
+					colors = {
+						"light_blue"
+						"purple"
+						"pink"
+						"yellow"
+					}
+				}
+				ignore_initial_colony_error = yes
+				species = event_target:hfe_egypt_species # As Thoth was once a Ju Ahet
+				day_zero_contact = no
+				effect = {
+					kill_leader = {
+						type = ruler
+					}
+					create_leader = {
+						name = "Thoth"
+						set_age = 3096
+						class = scientist
+						sub_type = society # Military theory
+						species = event_target:hfe_varelse_leftover_robots
+						event_leader = yes
+						gender = indeterminable
+						immortal = yes
+						skill = 10
+						traits = {
+							trait = leader_trait_righteous
+							trait = leader_trait_ruler_synthetic
+							trait = leader_trait_expertise_military_theory
+							trait = leader_trait_fleet_logistician
+							trait = leader_trait_cataloger
+#							trait = hfe_scientist_trait_wary
+						}
+					}
+					assign_leader = last_created_leader
+					if = {
+						limit = { has_global_flag = HFE_TSP_spawn_test }
+						log = "Thoth country initialized"
+					}
+					give_technology = { message = no tech = tech_jump_drive_1 }
```

Comment:
I dont see why in need tech

Reply:
Flavor and when you destroy it, respectively.

Reply:
Station have no drive, you cant get it

Reply:
Okay, done

Reply:
Its make no sense to give thoth tech, because debres spawns from predefined artist design

Reply:
If I knew how, I'd write a design for Thoth. I have no idea how that works, though.